icons/notebook-tabs.svg

```diff
@@ -0,0 +1,21 @@
+<svg
```

Comment:
Less easy legible than the other notebooks.

Reply:
> Less easy legible than the other notebooks.

@jguddas Yeah, made [`note`]`book-tabs` wider…